UPDATING.md

```diff
@@ -59,6 +59,26 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### Cli use exactly single character for short option
+
+Use exactly one single character for short option in Airflow cli, New commands are available according to the following table:
+
+| Old command                                        | New command                                       |
+|----------------------------------------------------|---------------------------------------------------|
+| ``airflow (dags|tasks|scheduler) [-sd, --subdir]`` | ``airflow (dags|tasks|scheduler) [-S, --subdir]`` |
+| ``airflow tasks test [-dr, --dry_run]``            | ``airflow tasks test [-dr, --dry_run]``           |
```

Comment:
How about -dr option here? It should be '-n' as well I guess.

Reply:
Good catch!, fix it